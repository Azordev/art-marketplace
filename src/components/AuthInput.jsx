import React from "react";

const AuthInput = (props) => (
  <div className="relative">
    <input
      className="block w-full px-4 py-3 mb-4 border border-1 border-black border-opacity-50 rounded-0 focus:ring focus:ring-blue-500 focus:outline-none placeholder-black text-2xl"
      {...props}
    />
  </div>
);

export default AuthInput;
