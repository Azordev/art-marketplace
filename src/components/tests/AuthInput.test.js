import React from "react";
import renderer from "react-test-renderer";
import AuthInput from "../AuthInput";

it("renders correctly", () => {
  const tree = renderer
    .create(<AuthInput type="text" name="test" placeholder="test" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
