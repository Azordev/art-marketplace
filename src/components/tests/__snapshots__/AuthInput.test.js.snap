// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="relative"
>
  <input
    className="block w-full px-4 py-3 mb-4 border border-1 border-black border-opacity-50 rounded-0 focus:ring focus:ring-blue-500 focus:outline-none placeholder-black text-2xl"
    name="test"
    placeholder="test"
    type="text"
  />
</div>
`;
